### ISSUE TITLE
evaluator training  parameter

### ISSUE THREAD

**Issue by wangxu-1** [HuskyInSalt/CRAG#41]
Hi,sorry to bother you， I would like to further confirm the parameter settings for the evaluator training, specifically the batch_size, num_epochs, and seed
